sys/net/gnrc/network_layer/ipv6/nib/nib.c

```diff
@@ -626,6 +626,15 @@ static bool _resolve_addr(const ipv6_addr_t *dst, gnrc_netif2_t *netif,
                           _nib_onl_entry_t *entry)
 {
     bool res = false;
+    if ((netif != NULL) && (netif->device_type == NETDEV_TYPE_SLIP)) {
+        /* XXX: Linux doesn't do neighbor discovery for SLIP so no use sending
+         * NS and since SLIP doesn't has link-layer addresses anyway we can just
```

Comment:
has -> have, and a comma after "anyway" (I think).

Reply:
Fixed